Tests/SonarQube.TeamBuild.PreProcessor.Tests/AnalysisConfigGeneratorTests.cs

```diff
@@ -80,7 +80,7 @@ public void AnalysisConfGen_AnalysisConfigDoesNotContainSensitiveData()
             // Sensitive values - should not be written to the config file
             cmdLineArgs.AddProperty(SonarProperties.DbPassword, "secret db password");
             cmdLineArgs.AddProperty(SonarProperties.DbUserName, "secret db user");
-            cmdLineArgs.AddProperty("sonar.license.secured", "secret license number");
+            cmdLineArgs.AddProperty("sonar.user.license.secured", "user input license");
```

Comment:
This should be in the "Public args" section above, not under the "Sensitive values" section